docs/journalist/starting_client.rst

```diff
@@ -1,16 +1,74 @@
-Starting the SecureDrop client
+Starting the SecureDrop Client
 ==============================
 
 .. include:: ../includes/top-warning.rst
 
+Double-click on the **SecureDrop Workstation Launcher** desktop shortcut to
```

Comment:
The icon is simply called "SecureDrop" now. The term "launcher", while still used internally, is not exposed to the user and should not appear in documentation. Instead, to the user, it is exposed as the "SecureDrop Preflight Updater". We went with "Preflight Updater" as a term that is more specific to the function ("why does SecureDrop need a launcher? what does God need with a starship?").

Reply:
fixed in 5eda33f

Reply:
:vulcan_salute: